core/src/test/java/org/springframework/security/access/intercept/aopalliance/MethodSecurityInterceptorTests.java

```diff
@@ -126,9 +127,9 @@ public void gettersReturnExpectedData() {
 	}
 
 	@Test
-	public void missingAccessDecisionManagerIsDetected() throws Exception {
-		this.interceptor.setAccessDecisionManager(null);
-		assertThatIllegalArgumentException().isThrownBy(() -> this.interceptor.afterPropertiesSet());
+	public void missingAccessDecisionManagerIsDetected() {
```

Comment:
Should the tests also verify that a null authorization manager is detected?

Reply:
These tests are in `AbstractSecurityInterceptorTests`.